recipes/openvr/all/conanfile.py

```diff
@@ -20,83 +28,95 @@ class OpenvrConan(ConanFile):
         "fPIC": True,
     }
 
-    exports_sources = ["CMakeLists.txt", "patches/**"]
-    generators = "cmake"
-    _cmake = None
+    @property
+    def _min_cppstd(self):
+        return 11
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "msvc": "14.1",
+            "gcc": "5",
+            "clang": "5",
+            "apple-clang": "5.1",
+        }
```

Comment:
Despite wrong msvc version, why do you restrict more than before? It was raisong for gcc < 5 only, so we know it works for Visual Studio 2015 for example. You should keep previous logic.

Reply:
ok then, I've reverted :)